### PR TITLE
Adding Persian Food Sentiment Dataset

### PR DESCRIPTION
**Pull request by asparius**
## Checklist for adding MMTEB dataset

Reason for dataset addition:

Persian Food Sentiment dataset from [paper](https://arxiv.org/abs/2005.12515) and I believe it is the first Persian dataset.
- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [ ] I have added points for my submission to the [POINTS.md](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) file.

### CODE REVIEW

File: mteb/tasks/Classification/fas/PersianFoodSentimentClassification.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        bibtex_citation="""
+        @article{ParsBERT,
+            title={ParsBERT: Transformer-based Model for Persian Language Understanding},
+            author={Mehrdad Farahani, Mohammad Gharachorloo, Marzieh Farahani, Mohammad Manthouri},
+            journal={ArXiv},
+            year={2020},
+            volume={abs/2005.12515}
+        }
+        """,
+        n_samples={"train": 56700, "validation": 6300, "test": 7000},
```
Comment (isaac-chung) [inline mteb/tasks/Classification/fas/PersianFoodSentimentClassification.py:39]: Per the checklist item `I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)`, could we reduce both the validation and test sets sizes to 2048?

File: mteb/tasks/Classification/fas/PersianFoodSentimentClassification.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        name="PersianFoodSentimentClassification",
+        description="Persian Food Review Dataset",
+        reference="https://hooshvare.github.io/docs/datasets/sa",
+        dataset={
+            "path": "asparius/Persian-Food-Sentiment",
+            "revision": "92ba517dfd22f6334111ad84154d16a2890f5b1d",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
```
Comment (isaac-chung) [inline mteb/tasks/Classification/fas/PersianFoodSentimentClassification.py:18]: Can include the validation set as well.

**Review by isaac-chung — COMMENTED** [review]
Looks good! Just a few small comments.

It seems like farsi has been covered by `MassiveIntentClassification`, so please add 2 pts for yourself and 2 pts for me for the review.

**Comment by isaac-chung** [comment]
Note the changed scoring system format in https://github.com/embeddings-benchmark/mteb/pull/438.
